protected/views/project/view/filters.php

```diff
@@ -20,47 +20,49 @@
 ]);
 
     ?>
-    <div class="count">
-        <?php
-        echo Icon::healthFacility() . ' ' . \Yii::t('app', 'Health Facilities');
-        echo Html::tag('em', count($data));
-        ?>
-    </div>
-    <div class="count">
-        <?php
-        echo Icon::contributors() . ' ' . \Yii::t('app', 'Contributors');
-        echo Html::tag('em', $project->contributorCount);
-        ?>
-    </div>
-    <div class="count">
-        <?php
-        echo Icon::sync() . ' ' . \Yii::t('app', 'Latest update');
-        /** @var HeramsResponseInterface $heramsResponse */
-        $lastUpdate = null;
-        foreach($data as $heramsResponse) {
-            $date = $heramsResponse->getDate();
-            if (!isset($lastUpdate) || (isset($date) && $date->greaterThan($lastUpdate))) {
-                $lastUpdate = $date;
+    
+        <div class="count">
+            <?php
+            echo Icon::healthFacility() . ' ' . \Yii::t('app', 'Health Facilities');
+            echo Html::tag('em', count($data));
+            ?>
+        </div>
+        <div class="count">
+            <?php
+            echo Icon::contributors() . ' ' . \Yii::t('app', 'Contributors');
+            echo Html::tag('em', $project->contributorCount);
+            ?>
+        </div>
+        <div class="count">
+            <?php
+            echo Icon::sync() . ' ' . \Yii::t('app', 'Latest update');
+            /** @var HeramsResponseInterface $heramsResponse */
+            $lastUpdate = null;
+            foreach ($data as $heramsResponse) {
+                $date = $heramsResponse->getDate();
+                if (!isset($lastUpdate) || (isset($date) && $date->greaterThan($lastUpdate))) {
+                    $lastUpdate = $date;
+                }
             }
-        }
-        echo Html::tag('em', $lastUpdate? $lastUpdate->diffForHumans() : \Yii::t('app', 'N/A'));
-        ?>
-    </div>
-    <?php
-        echo Html::a('Filters', '#', ['id' => 'filter-expand']);
-        $this->registerJs(<<<JS
-        $('#filter-expand').on('click', function() {
-            $(this).parent().toggleClass('expanded');
-        });
+            echo Html::tag('em', $lastUpdate? $lastUpdate->diffForHumans() : \Yii::t('app', 'N/A'));
+            ?>
+        </div>
+        
+        <?php
+            echo Html::a('Filters', '#', ['id' => 'filter-expand']);
+            $this->registerJs(<<<JS
+            $('#filter-expand').on('click', function() {
+                $(this).parent().toggleClass('expanded');
+            });
 
-JS
-        )
+    JS
+            )
 
-    ?>
+        ?>
     <div class="advanced">
         <div class="filter filter_search">
             <?= Icon::search(); ?>
-            <input id="search-filter">
+            <input id="search-filter" placeholder="Search">
```

Comment:
This is the only change in this file as far as I can tell. Revert the change to indenting please.